worldwind/src/commonMain/kotlin/earth/worldwind/draw/DrawableSurfaceShape.kt

```diff
@@ -139,32 +141,32 @@ open class DrawableSurfaceShape protected constructor(): Drawable {
                     program.enableOneVertexMode(false)
                     program.loadScreen(colorAttachment.width.toFloat(), colorAttachment.height.toFloat())
 
-                    dc.gl.vertexAttribPointer(0 /*pointA*/, 4, GL_FLOAT, false, 20, 0)
-                    dc.gl.vertexAttribPointer(1 /*pointB*/, 4, GL_FLOAT, false, 20, 40)
-                    dc.gl.vertexAttribPointer(2 /*pointC*/, 4, GL_FLOAT, false, 20, 80)
-                    dc.gl.vertexAttribPointer(3 /*vertexTexCoord*/, 1, GL_FLOAT, false, 20, 56)
+                    dc.gl.vertexAttribPointer(0 /*pointA*/, 4, GL_FLOAT, false, 40, 0)
+                    dc.gl.vertexAttribPointer(1 /*pointB*/, 4, GL_FLOAT, false, 40, 80)
+                    dc.gl.vertexAttribPointer(2 /*pointC*/, 4, GL_FLOAT, false, 40, 160)
+                    dc.gl.vertexAttribPointer(3 /*texCoord + lineWidth*/, 2, GL_FLOAT, false, 40, 96)
+                    dc.gl.vertexAttribPointer(4 /*color*/, 4, GL_FLOAT, false, 40, 104)
```

Comment:
???? u32 is enough 4, GL_UNSIGNED_BYTE, true

Reply:
done